ui/src/components/SessionProvider.tsx

```diff
@@ -35,43 +35,42 @@ export default function SessionProvider({
     };
 
     const loadSession = async () => {
+      const isAuthenticated = await getInfo()
+        .then(
+          () => true,
+          () => false
```

Comment:
what is this doing?

Reply:
I use promise old fashion way instead of try catch block. Happy to  change it if you like. 

`getInfo()` should be called before `clearSessionIfNecessary`. `clearSessionIfNecessary` depends on `getConfig`. In case there is a session in localStorage and authentication is required, but user has no valid http session, the `getConfig` call will fail with http code 401.

Reply:
let me change it